pandas/core/indexes/datetimes.py

```diff
@@ -93,6 +93,7 @@ class DatetimeIndex(DatetimeTimedeltaMixin):
         'infer' can be passed in order to set the frequency of the index as the
         inferred frequency upon creation.
     tz : pytz.timezone or dateutil.tz.tzfile
+        Timezone information object from time zone information files.
```

Comment:
Not sure what you mean by `from time zone information files`. And  better be consistent when using `timezone` and `time zone`, use just one.

Reply:
Both `pytz.timezone` and `dateutil.tz.tzfile` read time zone information from binary files here is a reference list for this information 
- [dateutil](https://github.com/dateutil/dateutil/blob/110a09b4ad46fb87ae858a14bfb5a6b92557b01d/dateutil/tz/tz.py#L386)
- [pytz](https://github.com/stub42/pytz/blob/7b1a844c8ecf2996142ac0eb32201b676e9dcb9a/src/pytz/__init__.py#L123)
- [datetime.tzinfo](https://github.com/python/cpython/blob/1b55b65638254aa78b005fbf0b71fb02499f1852/Lib/datetime.py#L1145)

Time zone information files are typically found under a directory with a name like `/usr/share/zoneinfo` for linux.

For the first `Timezone` I was referring to python object and the others is information only.

Reply:
Thanks for the information. I wouldn't expect pandas users to know about the details, and this description should help them know what to provide for this parameter. I think something like `Time zone object corresponding to the desired time zone. For example dateutil.tz.UTC.` should be more clear than mentioning the time zone files.